scripts/trigger_docker_build.sh

```diff
@@ -3,7 +3,7 @@
 _circle_token=$1
 _grafana_version=$2
 
-trigger_build_url=https://circleci.com/api/v1/project/grafana/grafana-docker/tree/master?circle-token=${_circle_token}
+trigger_build_url=https://circleci.com/api/v1/project/grafana/grafana-docker/tree/deploy?circle-token=${_circle_token}
```

Comment:
I guess this needs be changed as well before merge

Reply:
Yes, good catch!